### ISSUE TITLE
[RoboCop: Rogue City] fix not fully working?

### ISSUE THREAD

**Issue by premiumghoul** [RoseTheFlower/UltrawideIndex#109]
I'm using the fix for RoboCop: Rogue City at 3440x1440 and while the FoV is corrected in gameplay and the black bars are removed from dialogue, the scenes themselves are still cropped on the top and bottom. And when entering dialogue the FoV will rapidly change and then change back before the screen fades out. Also, if you pause during dialogue the game will suddenly increase the FoV to the correct value but when you unpause it goes back to being zoomed in.

This is my first time playing the game so I'm not sure if that's just how this game/fix is or if it worked differently before.

**Comment by RoseTheFlower** [comment]
Given your edit, what is the issue? Regarding the pause screens, the mod has a check for whether a scene is dialogue because it requires a different approach to FOV correction, so when you pause, the mod sees it as you being outside dialogue. It may feel weird but it should have no effect on what you see when you're actually playing the game.

**Comment by premiumghoul** [comment]
Yeah, sorry, I realised that most of what I wrote no longer applies once I'd spent more time using the fix. The only thing that stands out is the really quick zoom in and out that happens when you start dialogue.

[https://www.youtube.com/watch?v=yZfC-nc0Rlg](https://www.youtube.com/watch?v=yZfC-nc0Rlg)

**Comment by RoseTheFlower** [comment]
That doesn't look great but it's probably a side effect of the same check I talked about. When developing the mod, I think I struggled to find a condition that's specific to dialogue and that's the best I could find. Let's keep the issue open in case I do another deep dive later and manage to find something better one day.

**Comment by RoseTheFlower** [comment]
Try v1.2 and report back if you can.

**Comment by RoseTheFlower** [comment]
What's the simplest way to reproduce the issue? What store is your game on? Which version of the fix?

**Comment by ccs19** [comment]
Hey, little late, but I haven't played since I posted. Tried to keep my eye open while I've been playing today.

This is on Steam. Using version 1.3 from [this page.](https://github.com/RoseTheFlower/UltrawideIndex/releases/tag/robocoprc)

The game unfortunately doesn't have manual saving and it's hard to know when the issue will come up because it requires the game to go to "non-combat" mode, though the issue is always present in the police HQ. 

I've added two saves + SaveIndex.sav in case the index is needed.

[SaveGames.zip](https://github.com/user-attachments/files/21567377/SaveGames.zip)

The first save (Autosave97) has a several minutes worth of combat that leads into a non-combat section where you have to talk to an NPC. It's after talking to the NPC that the FOV reduces. 

The second save (Autosave98) is right after that and has a short sequence which leads to the hospital where the FOV fix is not applied.

Both saves are on Hard difficulty. Reducing it to easy should make the first save much faster to get the non-combat section.

**Comment by RoseTheFlower** [comment]
I loaded from Autosave98, killed the enemies, watched the heli cutscene, then loaded into the hospital. The vertical FOV matched that of 16:9 (set to 100 in both cases). I'm on GOG though, but I also have the game on Steam, so if you show me a few screenshots from the affected scenes so that we know we're talking about the same things, I can install it and check.

**Comment by epic-piano** [comment]
I'm getting black bars in every cutscene on a 5120x1440 monitor. 

Here's a screenshot in one of the very first cinematics, plus it won't let my FOV go higher than 100 (which is the vanilla maximum)

![Image](https://github.com/user-attachments/assets/0bfc2123-6192-4c9d-8fc0-78e6964b4d4c)

**Comment by RoseTheFlower** [comment]
Though I don't remember if the fix ever intended to remove letterboxing, its description says it removes _vertical_ black bars. If you click Detailed features, that reads 

> Developers' main view correction code replaced for scaling consistency. This eliminates the need to set your resolution and touch the FOV slider to achieve vertical parity at the menu and everywhere else. Note that the max FOV text will be capped at 100.
